oe_theme.libraries.yml

```diff
@@ -66,9 +43,3 @@ tables:
   dependencies:
     - core/drupal
     - oe_theme/base
-timelines:
```

Comment:
Library was remved but is still being loaded on timeline.ui_patterns.yml

Reply:
Very good catch.